### DISCUSSION TITLE
How to make this awesome nuxt module social card?

### DISCUSSION THREAD

**Discussion by kingyue737** [volkanakkus/nuxt-tradingview#5] [Q&A] [answered]
Sorry for asking a question not truly related to this project. I found many nuxt modules use this awesome template to make a social card. I'm also writing a nuxt module and would like to know how to generate a similar one. I really appreciate any help you can provide.

![cover](https://raw.githubusercontent.com/volkanakkus/nuxt-tradingview/assets/cover.png)

**Comment by volkanakkus** [comment, marked as answer]
Hello, yes, it's a bit of an unrelated topic, but that's okay. 

Actually, I couldn't find an online resource to create this kind of cover, so I created this visual myself by using existing covers and the [Nuxt Brand Kit ](https://nuxt.com/design-kit) Figma file.
